Update telemetry.rst

justinegeffen opened the pull request:
Updated Playbooks telemetry terminology and updated section title.

justinegeffen commented:
@agarciamontoro, I'm not sure whether I got the correct names for things so feel free to edit as needed. If there are new telemetry items, LMK, and I'll add them. :)

agarciamontoro reviewed (commented):
The identifiers of the telemetry fields have not changed. We did add new ones, but the old ones keep the same terminology.

I can update the list here if you want me to, the source of truth is this file: https://github.com/mattermost/mattermost-plugin-playbooks/blob/master/server/telemetry/rudder.go

justinegeffen commented on source/manage/telemetry.rst:
```diff
@@ -142,56 +142,56 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
+- ``NumChecklists``: Number of stages in this run.
+- ``TotalChecklistItems``: Number of tasks in this run.
+- ``ActiveStage``: A number indicating the stage of the run (0-based).
+- ``Action``: The type of action performed against the run: ``start``, ``end``, ``restart``, ``change_stage``, ``change_owner``, ``update_status``.
```
```suggestion
- ``Action``: The type of action performed against the run: ``create``, ``end``, ``restart``, ``change_stage``, ``change_commander``, ``update_status``, ``add_timeline_event_from_post``, ``update_retrospective``, ``publish_retrospective``, ``remove_timeline_event``.
```

justinegeffen commented on source/manage/telemetry.rst:
```diff
@@ -142,56 +142,56 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
 **Data collected in tasks events**
 
-- ``IncidentID``: Unique identifier of the incident.
+- ``runID``: Unique identifier of the run.
```
```suggestion
- ``IncidentID``: Unique identifier of the run.
```

justinegeffen commented on source/manage/telemetry.rst:
```diff
@@ -142,56 +142,56 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
+**Data collected in run events**
 
-- ``IncidentID``: Unique identifier of the incident.
+- ``runID``: Unique identifier of the run.
```
```suggestion
- ``IncidentID``: Unique identifier of the run.
```

justinegeffen commented on source/manage/telemetry.rst:
```diff
@@ -142,56 +142,56 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
 - ``ServerVersion``: Version of the server the plugin is running on.
 - ``UserID``: Unique identifier of the server.
-- ``Event``: Type of the event. There are three event types that are tracked: ``incident``, ``tasks``, ``playbook``.
+- ``Event``: Type of the event. There are three event types that are tracked: ``run``, ``tasks``, ``playbook``.
```
```suggestion
- ``Event``: Type of the event. There are three event types that are tracked: ``incident``, ``tasks``, ``playbook``.
```

justinegeffen commented on source/manage/telemetry.rst:
```diff
@@ -155,43 +155,43 @@ The following list details the types of Incident Collaboration metadata we colle
 - ``BroadcastChannelID``: Unique identifier of the channel where the status updates will be broadcasted.
 - ``UsesReminderMessageTemplate``: ``true`` if the playbook has a template for the reminder message, ``false`` otherwise.
-- ``Action``: The type of action performed against this playbook: ``create``, ``update``, ``delete``.
+- ``Action``: The type of action performed against this playbook: ``start``, ``update``, ``delete``.
```
```suggestion
- ``Action``: The type of action performed against this playbook: ``create``, ``update``, ``delete``.
```

justinegeffen commented:
Thanks, @agarciamontoro! I've reverted the ones I changed and also added updated info. I imagine I missed a few other updates - if you're open to adding whatever I've missed to this list that'd be awesome. :)

agarciamontoro commented on source/manage/telemetry.rst:
```diff
@@ -155,36 +155,36 @@ The following list details the types of Incident Collaboration metadata we colle
-- ``ActiveStage``: A number indicating the stage of the incident (0-based).
-- ``Action``: The type of action performed against the incident: ``create``, ``end``, ``restart``, ``change_stage``, ``change_commander``, ``update_status``.
-- ``Public``: When creating an incident, ``true`` if it is public, and ``false`` if it is private.
+- ``OwnerUserID``: Unique identifier of the owner of the run.
```
```suggestion
- ``CommanderUserID``: Unique identifier of the owner of the run.
```

agarciamontoro reviewed (commented):
@justinegeffen, sorry  but I haven't had time to add the new info here :( 

I left a single comment of one field that was still changed, but otherwise looks good.

What do you think if we merge this and iterate when I'm back from my PTO?

justinegeffen commented:
> What do you think if we merge this and iterate when I'm back from my PTO?

That works perfectly for me. Should I add it to the sprint board?

agarciamontoro reviewed (approved):
:tada:

agarciamontoro commented:
> That works perfectly for me. Should I add it to the sprint board?

Sounds good!
